index.css

```diff
@@ -96,3 +102,78 @@ main {
     margin: auto;
     text-align: center;
 }
+
+
+.wrapper {
+    margin: 0 auto;
+    background-color: #ffffff;
+    height: 50px;
+    padding: 0;
+
+
+
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+    border-radius: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: #ffffff;
+
+
+}
+
+header img.logo {
+    padding: 5px;
+    width: 40px;
+    height: 40px;
+}
+
+header img.profile {
+    width: 40px;
+    height: 40px;
+    padding: 5px;
+    border-radius: 50%;
+
+
+}
```

Comment:
To avoid the boiler-plate I would join two classes. Padding is also not needed anymore.

.img-logo, 
.img-profile {
    width: 100px;
    height: 100px;
}

.img-profile {
    border-radius: 50%;
}